README.md

```diff
@@ -334,37 +334,45 @@ bazel test //py:all
+| `bazel test //rb/...`                                                            | Run unit, all integration tests and lint           |
+| `bazel test //rb:lint`                                                           | Run RuboCop linter                                 |
+| `bazel test //rb/spec/...`                                                       | Run unit and integration tests for all browsers    |
+| `bazel test //rb/spec/... --test_size_filters small`                             | Run unit tests                                     |
+| `bazel test //rb/spec/unit/...`                                                  | Run unit tests                                     |
+| `bazel test //rb/spec/... --test_size_filters large`                             | Run integration tests for all browsers             |
+| `bazel test //rb/spec/integration/...`                                           | Run integration tests for all browsers             |
+| `bazel test //rb/spec/integration/... --test_tag_filters firefox`                | Run integration tests for local Firefox only       |
+| `bazel test //rb/spec/integration/... --test_tag_filters firefox-remote`         | Run integration tests for remote Firefox only      |
+| `bazel test //rb/spec/integration/... --test_tag_filters firefox,firefox-remote` | Run integration tests for local and remote Firefox |
+
+Ruby test targets have the same name as the spec file with `_spec.rb` removed, so you can run them individually.
+Integration tests targets also have a browser and remote suffix to control which browser to pick and whether to use Grid.
+
+| Test file                                               | Test target                                                      |
+| ------------------------------------------------------- | ---------------------------------------------------------------- |
+| `rb/spec/unit/selenium/webdriver/proxy_spec.rb`         | `//rb/spec/unit/selenium/webdriver:proxy`                        |
+| `rb/spec/integration/selenium/webdriver/driver_spec.rb` | `//rb/spec/integration/selenium/webdriver:driver-chrome`         |
+| `rb/spec/integration/selenium/webdriver/driver_spec.rb` | `//rb/spec/integration/selenium/webdriver:driver-chrome-remote`  |
+| `rb/spec/integration/selenium/webdriver/driver_spec.rb` | `//rb/spec/integration/selenium/webdriver:driver-firefox`        |
+| `rb/spec/integration/selenium/webdriver/driver_spec.rb` | `//rb/spec/integration/selenium/webdriver:driver-firefox-remote` |
 
 Supported browsers:
 
 * `chrome`
 * `edge`
 * `firefox`
+* `firefox-beta`
 * `ie`
-* `safari` (cannot be run in parallel - use `--local_test_jobs 1`)
-* `safari-preview` (cannot be run in parallel - use `--local_test_jobs 1`)
+* `safari`
+* `safari-preview`
 
 Useful command line options:
 
+* `--pin_browsers` - use browsers and drivers downloaded by Bazel
```

Comment:
the link in line 376 takes you up to this:
https://github.com/SeleniumHQ/selenium/pull/13683/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R240
which includes --pin_browsers but we can duplicate if we want

Reply:
Thanks for pointing that out, I'll update the README.